section2/exercises/if-statements.js

```diff
@@ -81,16 +110,29 @@ When cupsOfFlour = 1 and hasSauce = false, your program should log "I cannot mak
 When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pizza";
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
-
-var cupsOfFlour = 1;
+// Write your conditional here
+var cupsOfFlour = 8;
 var hasSauce = true;
 
-// Write your conditional here
+if (cupsOfFlour <= 1 && hasSauce === false) {
+  console.log(`I cannot make pizza`);
+} else if (cupsOfFlour >= 2 && hasSauce === true) {
+  console.log(`I can make pizza`);
+}
```

Comment:
Spend a little time reviewing this question now that you've been introduced to logical operators.

I'm moving pretty quick, but I believe the OR (`||`) logical operator would come in handy to catch more situations.